### ISSUE TITLE
Create phpStorm datasource configuration from local.xml

### ISSUE THREAD

**Issue by amenk** [enriquepiatti/Magicento#35]
What do you think about this one?

https://github.com/netz98/n98-magerun/issues/92

**Comment by amenk** [comment]
It would be nice if Magicento could create a datasource based on the current local.xml connection settings. It should be created in a way to only show and scan the current Magento database.

**Comment by jonpday** [comment]
:+1:  would love to see this feature

<edit>Changed my mind, sorry, don't think it is a big deal after all!</edit>

**Comment by amenk** [comment]
I asked on YouTrack if this is possible at all: http://youtrack.jetbrains.com/issue/IDEA-110475

**Comment by enriquepiatti** [comment]
Thanks Alex ! anyway I think is better to ask those things in their forum (http://devnet.jetbrains.com/community/idea/open_api_and_plugin_development) instead of opening an issue.
I've created the discussion for that don't worry (y)
http://devnet.jetbrains.com/thread/446518

**Comment by enriquepiatti** [comment]
I think this is not so easy, I couldn't find the extension point and there are too many variables to take into account, has the user the .jar for mysql? ssh settings? 
Personally I've found the data source from PhpStorm a bit buggy and slow so I never use it, I prefer to use special software like mysql workbench, sqlyog etc.
But I think I'm not understanding why this could be so important, I mean to configure the data source should take less than 20 or 30 seconds for project (and it's only done once). Am I loosing something here?

**Comment by jonpday** [comment]
I agree with @enriquepiatti it is a quick, once-off task to create a data source in PhpStorm and I would rather see other tickets be higher priority. I've also experienced PhpStorm's Database tools crash my whole OS which is a pity since they are much more functional than MySQL Workbench :-1:
